src/shared/actions/index.js

```diff
@@ -193,15 +199,13 @@ export function commitGroup(groupInfo: GroupInfo): ThunkAction {
         const firebase = getFirebase();
         const userId = firebase.auth().currentUser.uid;
         // get a single timestamp upon completion of the group
-        const timestamp = GLOBAL.DB.getTimestamp();
-        const endTime = timestamp;
+        const endTime = GLOBAL.DB.getTimestamp();
         const { groupId, projectId, results } = groupInfo;
         dispatch(startSendingResults(projectId, groupId));
         const { startTime, ...rest } = results[projectId][groupId];
         const objToUpload = {
             startTime,
             endTime,
-            timestamp,
```

Comment:
okay so from this release on there will be no `timestamp` attribute in the results anymore. I just double checked that this is supported by the back end. It's safe to remove here, the firebase function do not expect a timestamp attribute anymore.